apps/web-client/src/routes/orders/suppliers/[id]/+page.svelte

```diff
@@ -207,6 +217,37 @@
 						</table>
 					</div>
 				</div>
+
+				<div class="w-full">
+					<h2 class="text-lg">Other Supplier Publishers</h2>
+					<div class="w-full rounded border border-gray-200">
+						<table class="!my-0 flex-col items-stretch overflow-hidden">
+							<thead>
+								<tr>
+									<th scope="col" class="px-2 py-2">Publisher name</th>
+								</tr>
+							</thead>
+							<div class="h-max-[164px] w-full overflow-y-auto">
+								<tbody>
+									{#each publishersAssignedToOtherSuppliers as publisher}
+										<tr class="hover focus-within:bg-base-200">
+											<td class="px-2">{publisher}</td>
+											<td class="px-2 text-end"
+												><button
+													on:click={() => {
+														confirmationPublisher = publisher;
+														confirmationDialogOpen.set(true);
```

Comment:
When you resolve conflicts, there will be an error (probably coming from updated deps) stating that `div`, can't be nested under `table`, `thread`, `tbody`, etc. 
You can use the other two tables in `main` as a template (it was annoying and painful)